src/anybuf.rs

```diff
@@ -879,16 +881,27 @@ mod tests {
         // echo "id: \"no strings\"; strings: []" | protoc --encode=Collection *.proto | xxd -p -c 9999
         let data = Anybuf::new()
             .append_string(1, "no strings")
-            .append_repeated_string(9, &[]);
+            .append_repeated_string::<&str>(9, &[]);
         assert_eq!(data.into_vec(), hex!("0a0a6e6f20737472696e6773"));
+
+        // vector of Strings
+        // echo "id: \"strings\"; strings: [\"a\", \"b\", \"c\"]" | protoc --encode=Collection *.proto | xxd -p -c 9999
+        let owned = vec!["a".to_string(), "b".to_string(), "c".to_string()];
+        let data = Anybuf::new()
+            .append_string(1, "strings")
+            .append_repeated_string(9, &owned);
+        assert_eq!(
+            data.into_vec(),
+            hex!("0a07737472696e67734a01614a01624a0163")
+        );
     }
 
     #[test]
     fn append_repeated_bytes_works() {
         // echo "id: \"bytess\"; bytess: [\"\", \"a\", \"bcde\"]" | protoc --encode=Collection *.proto | xxd -p -c 9999
         let data = Anybuf::new()
             .append_string(1, "bytess")
-            .append_repeated_bytes(10, &[b"", b"a", b"bcde"]);
+            .append_repeated_bytes::<&[u8]>(10, &[b"", b"a", b"bcde"]);
```

Comment:
String slices could also be converted to bytes
```suggestion
            .append_repeated_bytes(10, &["", "a", "bcde"]);
```